examples/add_pyramid.py

```diff
@@ -0,0 +1,32 @@
+"""
+Displays an image pyramid
+"""
+
+from skimage import data
+from skimage.color import rgb2gray
+from skimage.transform import pyramid_gaussian
+import napari
+from napari.util import app_context
+import numpy as np
+
+
+image = data.astronaut()
+rows, cols, dim = image.shape
+
+
+# create pyramid from astronaut image
+astronaut = rgb2gray(data.astronaut())
+base = np.tile(astronaut, (16, 16))
+pyramid = list(pyramid_gaussian(base, downscale=2, multichannel=False))[:-8]
```

Comment:
I think it's nicer to use the `max_layer` kwarg here?

Reply:
done